### ISSUE TITLE
Flipped layout

### ISSUE THREAD

**Issue by khalidchawtany** [kdheepak/tabline.nvim#13]
Can we have a flipped layout option such that tabs appear on the right and the buffers appear on the left.

**Comment by kdheepak** [comment]
Yes, it would be nice to have this. Once I work on a better integration with lualine such that it works in any section, I imagine this issue will be resolved.

**Comment by kdheepak** [comment]
If you are interested in just the default buffer and tab as components in the tabline, I’d recommend using `lualine`’s built in buffer and tabs. They have been merged into shadmansaleh’s fork of lualine:
 https://github.com/shadmansaleh/lualine.nvim/pull/42
